io-engine/src/bdev/nvmx/handle.rs

```diff
@@ -576,6 +577,11 @@ impl BlockDeviceHandle for NvmeDeviceHandle {
             });
         }
 
+        let flags = mode.map_or(self.prchk_flags, |m| match m {
+            ReadMode::Normal => 0,
+            ReadMode::UnwrittenFail => SPDK_NVME_IO_FLAGS_UNWRITTEN_READ_FAIL,
+        });
```

Comment:
Now prchk_flags are not passed at all now, is this ok? Should prchk_flags be merged with mode?

Reply:
Fixed